pkg/cloudprovider/vsphere/instances_test.go

```diff
@@ -22,7 +22,7 @@ import (
 	"testing"
 
 	"github.com/vmware/govmomi/simulator"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit: is this change needed ?

Reply:
The go lint seems to have added that in to remove ambiguity. Each time I try to save the file without it, it adds it back in.